public/react-services/generic-request.js

```diff
@@ -28,18 +28,23 @@ export class GenericRequest {
       const { timeout } = wazuhConfig.getConfig();
       const requestHeaders = {
         'Content-Type': 'application/json',
-        'kbn-xsrf': 'kibana'
+        'kbn-xsrf': 'kibana',
       };
       const tmpUrl = getHttp().basePath.prepend(path);
 
-      try{
-        requestHeaders.pattern = (await getDataPlugin().indexPatterns.get(AppState.getCurrentPattern())).title;
-      }catch(error){};
+      try {
+        requestHeaders.pattern = (
+          await getDataPlugin().indexPatterns.get(AppState.getCurrentPattern())
+        ).title;
+      } catch (error) {
+        throw new Error('Error obtaining the current index patten', error);
```

Comment:
**issue**: the `Error` class only receive an argument in its `constructor`

**suggestions: Remove the second argument or integrate the error variable/message in the first argument.